.github/workflows/build-openstack-capi-image.yml

```
@@ -0,0 +1,85 @@
+name: Build OpenStack VM CAPI image
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Kubernetes version
+        required: true
+        type: string
+      tag:
+        description: ck8s-capi tag
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        description: Kubernetes version
+        required: true
+        type: string
+      tag:
+        description: ck8s-capi tag
+        required: true
+        type: string
+
+env:
+  version: ${{ inputs.version }}
+  tag: ${{ inputs.tag }}
+
+
+defaults:
+  run:
+    working-directory: ./images/capi
+
+jobs:
+  build-image:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v5
+        name: Checkout
+
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.config/packer/plugins
```

Question: Any reason this is multiline? Wouldn't this work?

```suggestion
          path: ~/.config/packer/plugins
```

---

I don't remember exactly, should be changed. Will look at it.